cucumber/authenticators_azure/features/authn_status_azure.feature

```diff
@@ -0,0 +1,161 @@
+        id: provider-uri
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+      - !group
+        id: operators
+        annotations:
+          description: Group of users who can check the status of the authn-azure/prod authenticator
+
+      - !permit
+        role: !group operators
+        privilege: [ read ]
+        resource: !webservice status
+
+    - !user alice
+
+    - !grant
+      role: !group conjur/authn-azure/prod/users
+      member: !user alice
+
+    - !grant
+      role: !group conjur/authn-azure/prod/operators
+      member: !user alice
+    """
+    And I am the super-user
+    And I successfully set Azure variables
+    And I login as "alice"
+    When I GET "/authn-azure/prod/cucumber/status"
+    Then the HTTP response status code is 200
+    And the authenticator status check succeeds
+
+  Scenario: A non-responsive Azure AD provider returns a 500 response
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-azure/prod
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for prod, based on Open ID Connect.
+
+      - !webservice
+        id: status
+        annotations:
+          description: Status service to verify the authenticator is configured correctly
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: id-token-user-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+      - !group
+        id: operators
+        annotations:
+          description: Group of users who can check the status of the authn-azure/prod authenticator
+
+      - !permit
+        role: !group operators
+        privilege: [ read ]
+        resource: !webservice status
+
+    - !user alice
+
+    - !grant
+      role: !group conjur/authn-azure/prod/users
+      member: !user alice
+
+    - !grant
+      role: !group conjur/authn-azure/prod/operators
+      member: !user alice
+    """
+    And I am the super-user
+    And I successfully set Azure provider-uri variable to value "https://not-responsive.com"
+    And I login as "alice"
+    When I GET "/authn-azure/prod/cucumber/status"
+    Then the HTTP response status code is 500
+    And the authenticator status check fails with error "ProviderDiscoveryFailed: CONJ00011E"
+
+  Scenario: provider-uri variable is missing and a 500 error response is returned
+    Given a policy:
+     """
+    - !policy
+      id: conjur/authn-azure/prod
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for prod, based on Open ID Connect.
+
+      - !webservice
+        id: status
+        annotations:
+          description: Status service to verify the authenticator is configured correctly
+
+      - !variable
+        id: id-token-user-property
```

Comment:
why do we need this for authn-azure?

Reply:
shouldnt this be `provider-uri`?

Reply:
to mock some dummy variable that is not the `provider uri`? If so, I would just not add any value b/c that is confusing because we also have OIDC that actually uses this value

Reply:
nice catch. missed it while copying this policy from OIDC.